resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/config.scala

```diff
@@ -52,6 +52,14 @@ package object config extends Logging {
       .timeConf(TimeUnit.MILLISECONDS)
       .createOptional
 
+  private[spark] val AM_CLIENT_MODE_EXIT_DIRECTLY =
+    ConfigBuilder("spark.yarn.am.clientModeExitDirectly")
+      .doc("When ture, if YarnClientSchedulerBackend.MonitorThread got report with " +
```

Comment:
nit: s/got/gets/

Reply:
true is misspelled, perhaps rename to ClientModeExitOnError, I don't think user case about MonitorThread, I think we should state this more that it will system exit, but note that if this is true can called from another application it  will terminate the parent application as well.

Reply:
> true is misspelled, perhaps rename to ClientModeExitOnError, I don't think user case about MonitorThread, I think we should state this more that it will system exit, but note that if this is true can called from another application it will terminate the parent application as well.

Nice suggestion,  Your consideration is very comprehensive， updated